src/main/java/seedu/address/model/task/ReadOnlyTask.java

```diff
@@ -71,18 +76,21 @@ default String getAsText() {
         buildDeadlineString(builder);
         buildStartEndDateTimeString(builder);
         buildTagsString(builder);
+        buildDoneString(builder);
 
         return builder.toString();
     }
 
     default void buildNameString(final StringBuilder builder) {
         builder.append(getName());
+        builder.append("\n");
     }
 
     default void buildDeadlineString(final StringBuilder builder) {
         // TODO don't include milliseconds in toString output
-        builder.append(" Deadline: ");
+        builder.append("Deadline: ");
         builder.append(getDeadline().isPresent() ? getDeadline().get().toString() : "none");
+        builder.append("\n");
```

Comment:
If there are too many lines will this approach be good? Sure it's better than vertical scrolling but having the ResultDisplay wrap is a more important step than doing this which requires more thought?